api/src/main/resources/liquibase-update-to-2.1.xml

```diff
@@ -35,5 +35,226 @@
+			<column name="start_date" type="datetime"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201610242135-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(*) from cohort_member
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<update tableName="cohort_member">
+			<column name="start_date" type="DATETIME" valueComputed="(select date_created from cohort)"/>
+			<where>cohort_id in (select cohort_id from cohort)</where>
+		</update>
+	</changeSet>
+	<changeSet author="vshankar" id="201610242135-3.1">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="cohort_member" columnName="start_date"/>
+		</preConditions>
+		<addNotNullConstraint tableName="cohort_member" columnName="start_date" columnDataType="DATETIME"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-4">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="end_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="end_date" type="datetime"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="creator"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="creator" type="int"/>
+		</addColumn>
+		<addForeignKeyConstraint baseColumnNames="creator" baseTableName="cohort_member" constraintName="cohort_member_creator" deferrable="false" initiallyDeferred="false" referencedColumnNames="user_id" referencedTableName="users"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5.1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(*) from cohort_member
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<update tableName="cohort_member">
+			<column name="creator" valueNumeric="1" type="int"/>
+		</update>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5.2">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="cohort_member" columnName="creator"/>
+		</preConditions>
+		<addNotNullConstraint tableName="cohort_member" columnName="creator" columnDataType="int"/>
+	</changeSet>
+	<changeSet id="201610042145-1" author="vshankar">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_created"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="date_created" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-1.1" author="vshankar">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					SELECT COUNT(*) from cohort_member
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<update tableName="cohort_member">
+			<column name="date_created" type="DATETIME" valueComputed="(select date_created from cohort)"/>
```

Comment:
I'm updating the date_created column of cohort_member with the value from the cohort table.